README.md

```diff
@@ -58,123 +58,130 @@ See [contributing](CONTRIBUTING.md) for more information.
+
+- Able to query liturgical dates for any year in the gregorian calendar (1582 - now). Note that dates for years before 1969 will still be returned in a format conforming to [Mysterii Paschalis](http://www.romcal.net/mysterii.html) even though those years came before the calendar reforms in 1969.
+- Filter queries to allow more streamlined date results to be obtained for the year.
+- Localization of liturgical date names to cater for different languages.
+- National liturgical calendars for country specific calendars.
+- Richly commented code to help developers and contributors understand how the module works.
 
 NOTE: This module uses [Moment.js](http://momentjs.com/) and [Lodash](http://lodash.com/) for calculations and operations. Additional plugins such as [Range](https://github.com/gf3/moment-range) and [Recur](https://github.com/c-trimm/moment-recur) are used to extend date computation functionality.
 
 ## Module Robustness & Data Integrity <a name="disclaimer"></a>
-*Calendar entries for this module are pulled from various sources from the net. As such their accuracy cannot be ensured. If you find an incorrect calendar entry (e.g. wrong date, wrong feast type, spelling issue, typos), you are most welcome to contribute to the source code or inform me so that the necessary changes can be made to make this a more robust and reliable module*
 
-*romcal's code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal's output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub.*
+_Calendar entries for this module are pulled from various sources from the net. As such their accuracy cannot be ensured. If you find an incorrect calendar entry (e.g. wrong date, wrong feast type, spelling issue, typos), you are most welcome to contribute to the source code or inform me so that the necessary changes can be made to make this a more robust and reliable module._
+
+_romcal’s code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal’s output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub._
 
 ### Testing romcal
+
 romcal employs `TDD` using `mocha` and `should`.
 Run `npm test` in your console to view test output.
 
-`Travis CI` is used to validate romcal builds to ensure functionality is working as expected.
+Travis CI is used to validate romcal builds to ensure functionality is working as expected.
 
 ## Builds <a name="builds"></a>
 
-Running `npm run build` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
+Running `npm install` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
```

Comment:
I think this section could be rewritten with something like that:

---
The romcal source code is actually the same for backend and frontend usage. To make it working on the frontend side, the frontend library is wrapped and built thanks to `webpack`. This build step is done automatically when you perform a `npm install` on the romcal directory.

But if you have edited the codebase, you can update manually the frontend library  with this command:
```
npm run build
```
The frontend library is generated in the `dist` folder, not included in the romcal codebase:
- `dist/romcal.bundle.min.js` A minified and obfruscated bundle of romcal + all its dependencies wrapped in a UMD module shell that is suitable for being included directly in browsers

Reply:
Okay, I change this, but I’d suggest to merge last two paragraphs (including the bulleted one) into one paragraph like this:

```bash
The frontend library is generated in the `dist` folder, not included in the romcal codebase. When built, it contains `romcal.bundle.min.js`, a minified and obfruscated bundle of romcal + all its dependencies wrapped in a UMD module shell that is suitable for being included directly in browsers.
```